### ISSUE TITLE
scope of map function is not specificed

### ISSUE THREAD

**Issue by happyshows** [Appsilon/shiny.collections#8]
```
Warning: Error in empty_tibble: could not find function "map"
Stack trace (innermost first):
    46: empty_tibble
    45: cursor_to_tibble
    44: shiny::reactiveValues
    43: shiny.collections::collection
    42: server [/Users/showsle/R/Atwitter/server.R#2]
     1: runApp
Error in empty_tibble(column_names) : could not find function "map"
```

**Comment by happyshows** [comment]
I'm guessing it's from tidyverse?  Could you put :: before the function?

**Comment by dokato** [comment]
Yes, thank you for pointing this out. Now it's fixed (PR #10 ).
